### ISSUE TITLE
[1.11.2] No default structures loading in SMP

### ISSUE THREAD

**Issue by ArmandVestibule** [Ivorforce/RecurrentComplex#154]
Please see Issue #63. I figured I would start a new issue instead of bringing that one back from the dead.

In any event, I am now on the latest version of the mod and its dependencies, residing on a 1.11.2 SMP server. The issues I am experiencing are identical to the ones I detailed in the original issue, even to this day and with a wildly different list of mods.

As per that issue, no structures generate and /#list comes up empty. This occurs even if ReC/IvToolkit are the only mods installed and running.

![image](https://cloud.githubusercontent.com/assets/10107523/22230927/74197bd8-e1ae-11e6-8b6b-b389217db909.png)

Output of /#list is as per the above image. I have tried this in both standard configuration and in lite mode, to no avail.

I am currently running forge-1.11.2-13.20.0.2224.

**Comment by Ivorforce** [comment]
Can you post the full log of that (or any other) session where you use /#reload MODDED, as well as /#reload CUSTOM)

**Comment by ArmandVestibule** [comment]
You want client, server or both?

**Comment by Ivorforce** [comment]
Server only

**Comment by ArmandVestibule** [comment]
Roger that. Starting from a fresh boot -- will have it in a few minutes.

**Comment by ArmandVestibule** [comment]
Attached. I invoked the commands from a player session at 09:58:20. I provided both logs just in case.

[ReC-fml-server-latest.txt](https://github.com/Ivorforce/RecurrentComplex/files/727023/ReC-fml-server-latest.txt)

**Comment by Ivorforce** [comment]
The problem is that Recurrent Complex is getting an 'Access Denied' when trying to read the files. I'm not sure how that is possible, considering textures etc. are being read out, but it might have to do with the fact that ReC uses a newer, and thus different Java API for reading files than MC does.

I recommend checking and updating the permission levels on the mod files to be readable by all users and programs.

**Comment by ArmandVestibule** [comment]
I thought the same thing when I first scoured these logs yesterday. The server process has full control over the entire directory already, to no avail. I even went as far as opening up the entire mods folder to Everyone but this error reoccurs with every mod in the mods folder regardless of that setting.

To try and circumvent this issue, I will try running the process as an elevated account to see what happens and get back  to you.

Is there anything else you might suggest while I do that?

**Comment by Ivorforce** [comment]
It's possible that it somehow doesn't find the files. Maybe in your file path there is a special character or something? I mean, your problem is _very_ limited to you (or very few others that don't report issues).

**Comment by ArmandVestibule** [comment]
Okay.  Well, following some tinkering it appears that I have found the root cause. This entire time, I had the server running under Network Service instead of the Local System. This had no apparent effect on any mods but ReC but as soon as I changed it over to a System process, everything in the mod registered and starting working.

In essence, definitely appeared to be a filesystem permissions issue and while it's mildly embarassing, it appears to be the root cause.

That being said, thanks for the help!
